themes/bootstrap3/templates/layout/layout.phtml

```diff
@@ -1,11 +1,12 @@
-<?=$this->doctype('HTML5')?>
+<?=$this->doctype()?>
+
 <html lang="<?=$this->layout()->userLang?>"<?php if ($this->layout()->rtl): ?> dir="rtl"<?php endif; ?>>
   <head>
     <?php $this->setupThemeResources(); ?>
-    <meta charset="utf-8">
```

Comment:
Oh, one last question: what's the motivation in removing this?

Reply:
Validator complained that you shouldn't have it when you already have `<meta http-equiv="Content-Type" content="text/html; charset=UTF-8">`. And that's what headMeta adds.

Reply:
Okay, that makes perfect sense!